dmscripts/framework_utils.py

```diff
@@ -0,0 +1,9 @@
+from dmapiclient import HTTPError
+
+
+def set_framework_result(client, framework_slug, supplier_id, result, user):
+    try:
+        client.set_framework_result(supplier_id, framework_slug, result, user)
+        return"  Result set OK: {} - {}".format(supplier_id, "PASS" if result else "FAIL")
```

Comment:
I'm a bit surprised the parser accepts this but I think the convention is to have a space after `return`